test/standalone/test-index.js

```diff
@@ -121,4 +122,43 @@ describe('index.js', function() {
     assert.equal(typeof agent.addTransactionLabel, 'function');
   });
 
+  it('should allow start and end span calls when disabled', function() {
+    agent.stop();
+    var span = agent.startSpan();
```

Comment:
Yea, there are a lot of layers of the api. Hopefully some good documentation and examples in the readme will clear up some of the confusion!